docs/pages/blog/aggregation-functions.md

```diff
@@ -0,0 +1,112 @@
+
+How easy can it be to get the average of transactions per user?
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/aggregation-functions/with-row-grouping.mp4" type="video/mp4" />
+</video>
+
+## Customize everything
+
+The out-of-the-box experience is very important to us, but as is the ability to customize and extend the feature to your own use case.
+
+You can change the styles of every element involved, set automatic aggregations, choose which columns can be aggregated by which functions, and even create your own custom function, like in the example below, where we created a function to get the first value in alphabetical order.
+
+```ts
+const firstAlphabeticalAggregation: GridAggregationFunction<string, string | null> =
+  {
+    apply: (params) => {
+      if (params.values.length === 0) {
+        return null;
+      }
+      const sortedValue = params.values.sort((a = '', b = '') => a.localeCompare(b));
+      return sortedValue[0];
+    },
+    // The `label` defines what's displayed in the column header when this
+    // aggregation is active.
+    label: 'First Alphabetical',
+    // The `types` property defines which type of columns can use this
+    // aggregation function. Here, we only want to propose this aggregation
+    // function for `string` columns. If not defined, aggregation will be
+    // available for all column types.
+    columnTypes: ['string'],
+  };
+```
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/aggregation-functions/with-custom-functions.mp4" type="video/mp4" />
+</video>
+
+## Okay, I’m in! How can I get started?
+
+:::info
+First make sure that you're using `@mui/x-data-grid-premium` and version `5.15.0` or later.
+:::
+
+During the initial version of Aggregation functions, we’re publishing the feature as experimental, so we have some room to validate its API and the overall direction we’re taking with it.
+But the feature is stable, and the API will likely not change significantly.
+
+To activate the feature, please enabled it with the experimentalFeatures prop.
+
+```tsx
+<DataGridPremium experimentalFeatures={{ aggregation: true }} {...otherProps} />
+```
+
+And that's it! That enables the aggregation option on the column menu (available by default on columns of type number and string).
+
+## Use with tree data and more
+
+We already mentioned that you could customize functions and use aggregation with row grouping. But there are a lot of other possibilities to explore!
+
+Please check out the feature’s [full documentation](/x/react-data-grid/aggregation/) to get a better overview of everything it has to offer.
+
+## Row pinning
+
+Aggregation footer row is empowered by another new feature called [Row pinning](/x/react-data-grid/rows/#row-pinning).
```

Comment:
What does "empowered" mean in this context? "Made possible by"? In that case, I'd just say "powered."

```suggestion
The Aggregation footer row is powered by another new feature called [Row pinning](/x/react-data-grid/rows/#row-pinning).
```